### ISSUE TITLE
Telegram crash when switching video to Picture-in-Picture on KDE

### ISSUE THREAD

**Issue by mokazemi** [telegramdesktop/tdesktop#25472]
### Steps to reproduce

1. Play a video
2. Click on Picture-in-Picture icon
3. It crashes!

### Expected behaviour

It should play video without any problem. It used to work before.

### Actual behaviour

It just crashes for a few seconds and then telegram closes. 

### Operating system

Arch Linux

**telegram-desktop** from Arch linux repos. 
**Desktop:** KDE Plasma 5.26.3, X11

### Version of Telegram Desktop

4.3.4

### Installation source

Other (unofficial) source

### Logs

```shell
When running telegram-desktop in terminal:

.
.
.
qt.svg: Error while inflating gzip file: SVG format check failed
qt.svg: Error while inflating gzip file: SVG format check failed
qt.svg: Error while inflating gzip file: SVG format check failed
QTextCursor::setPosition: Position '8388607' out of range
QTextCursor::setPosition: Position '8388607' out of range
Segmentation fault (core dumped)
```

**Comment by ilya-fedin** [comment]
There's a regression in Qt 6.4.1, switch to one of the official builds (from https://desktop.telegram.org), they have a fix

**Comment by svenstaro** [comment]
@ilya-fedin could you post the upstream ticket in Qt for that? We might be able to patch our Qt.

**Comment by svenstaro** [comment]
Are you generally interested in upstreaming your Qt patches by the way? It would to me to be quite the hassle to maintain a patch stack as large as tdesktop's indefinitely.

**Comment by ilya-fedin** [comment]
I think you need only that patch? No, I don't think I will upstream this patch, last time I tried to fix a crash started happening with v6.4.0, threy required me to provide a minimum reproducible example and sadly I can't create one, so I abandoned my change, but fortunately by that time @john-preston already made another fix in tdesktop code. If this won't happen that time and no one else would report/fix the bug, the patch is likely to stay indefinitely.

**Comment by ilya-fedin** [comment]
That may sound ridiculously, but we sadly just don't have enough manpower to conform to Qt's reporting rules. We even have an issue with a list of issues happening due to Qt bugs, so maybe anyone from the community would spend time on writing a minimum reproducible example (#25126), but no issue was fixed during this experiment.
